stdlib/public/runtime/CygwinPort.cpp

```diff
@@ -47,7 +47,7 @@ int swift::_swift_dl_iterate_phdr(int (*Callback)(struct dl_phdr_info *info,
   BOOL ret = EnumProcessModules(procHandle, modules.data(),
                                 modules.size() * sizeof(HMODULE), &neededSize);
 
-  if (ret == FALSE) {
```

Comment:
#define FALSE   0
#define TRUE    1

They are defined at /usr/include/w32api/ntdef.h in Cygwin.